sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -51,6 +51,7 @@ trait RunnableCommand extends Command {
 }
 
 trait LeafRunnableCommand extends RunnableCommand with LeafLike[LogicalPlan]
+trait UnaryRunnableCommand extends RunnableCommand with UnaryLike[LogicalPlan]
```

Comment:
Because `RunnableCommand` is defined in the module `sql/core`, it seems that adding `UnaryRunnableCommand` here is more appropriate.